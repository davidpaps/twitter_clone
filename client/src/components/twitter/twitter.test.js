import React from "react";
import { shallow } from "enzyme";
import Twitter from "./twitter";
import renderer from "react-test-renderer";

describe("Twitter", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<Twitter />)));

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(2);
  });

  it("should render a <img />", () => {
    expect(wrapper.find("img").length).toEqual(2);
  });

  it("should render correctly", () => {
    const tree = renderer.create(<Twitter />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
