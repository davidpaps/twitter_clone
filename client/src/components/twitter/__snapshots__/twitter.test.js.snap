// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Twitter should render correctly 1`] = `
Array [
  <div
    className="Title"
  >
    <img
      alt=""
      src="https://mcansozeri.files.wordpress.com/2013/07/twitter_logo.jpg"
    />
  </div>,
  <div
    className="Twitter"
  >
    <img
      alt=""
      src="https://webstockreview.net/images/lightbulb-clipart-self-realization-7.gif"
    />
    
  </div>,
]
`;
